hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/NetworkServiceImplTest.java

```diff
@@ -39,7 +39,7 @@ class NetworkServiceImplTest {
     private NetworkServiceImpl subject;
 
     @BeforeEach
-    void setUp() {
+    void before() {
```

Comment:
Does checkstyle complain about this?  `before` is fine, but IMO `setUp` is fine too

Reply:
All of these minor typos and annoying changes were reported from `checkstyle` tool. I just saw that those are not requirements for compilation and removed them!

Reply:
Ah I see, sounds good. Thanks!